spk/owfs/src/installer.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Package
+PACKAGE="owfs"
+DNAME="Owfs"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+SERVICETOOL="/usr/syno/bin/servicetool"
+FWPORTS="/var/packages/${PACKAGE}/scripts/${PACKAGE}.sc"
+
+CFG_FILE="${INSTALL_DIR}/var/owfs.conf"
+
+preinst ()
+{
+ #     if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+ #       if [ ! -d "${wizard_OWdriver}" ]; then
+ #           echo "OW hardware not set."
+ #           exit 1
+ #       fi
+#	fi
+
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        # Edit the configuration according to the wizard
+        sed -i -e "s|@OW_HARDWARE@|${wizard_OWdriver:=FAKE \= DS18S20,DS2405}|g" ${CFG_FILE}
+	fi
```

Comment:
Fix indent.

For the record, what is accepted as input for that field? Where do people get the driver and the name, seeing as it's not included in the package?